sdk/cosmos/src/clients/user_client.rs

```diff
@@ -135,21 +158,6 @@ impl UserClient {
         PermissionClient::new(self, permission_name)
     }
 
-    pub(crate) fn prepare_request_with_user_name(
```

Comment:
I think we actually want to keep this and just make sure that `replace_user` is also using it.

Reply:
Good catch, I pushed a change that adds it back and uses it in Get, Replace, and Delete